documentation/experimental_ui.md

```diff
@@ -0,0 +1,78 @@
+# Experimental UI
+---
+
+We believe that web scraping is a process. It might seem easy to extract first 
+data items, however we believe that the data delivery requires a bit more efforts or
+a process which supports it!
+
+Our aim is to provide you with the following services:
+
+1. Schedule (start and stop) your spiders on a cloud
+2. View running jobs (performance based analysis)
+3. View and validate scraped items for quality assurance and data analysis purposes.
+4. View individual items and compare them with the actual website.
+
+## Project status
+
+Currently the project is in early alpha stage. We're constantly working on making
+it more stable. And currently running it for long running jobs.
```

Comment:
~~And currently running it for long running jobs.~~

Reply:
~~so far it did no show major problems, however we accept that there will be problems on such early stages! If you are one of those who has them, please don't 
hesitate to report them here.~~

should report it on the crawly_ui repo, not crawly.

Reply:
ok, agree